.gitignore

```diff
@@ -21,4 +21,5 @@
 # Envs
 configs/driver.yaml
 .env*
-resolvers.settings.yaml
```

Comment:
why we removed resolvers.settings.yaml from gitignore ?

Reply:
Was something that was changed because at the beginning I had `signers.settings.yaml` and changed .gitignore with `*.settings.yaml`.
Updated again to `resolvers.settings.yaml`